### PR TITLE
indexing document store with Q&A pairs

### PR DESCRIPTION
**Pull request by murielhol**
The Makefile describes how to run everything, but first:

Make sure to have docker up and running, and to have run docker-compose pull beforehand.
setup the aws cli as described in the SETUP_S3_STORAGE.MD file
The access key and id I should have shared with you

### CODE REVIEW

File: document_indexing/populate_document_store.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import logging
+
+from haystack.document_stores import ElasticsearchDocumentStore
+from s3_storage import S3Storage
+# from rest_api.rest_api.schema import QuestionAnswerPair
```
Comment (rafaljanwojcik) [inline document_indexing/populate_document_store.py]: Why is everything commented out? It looks really nice, I think it's ok

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,16 @@
+all: elastic_search_up index_documents give_indexing_some_time rest_api_up run_query
+
+elastic_search_up:
+	docker-compose up elasticsearch
```
Comment (rafaljanwojcik) [inline Makefile]: shouldn't it also have the -d argument, as in docker-compose for haystack api? I think it ensures that the docker is running in the background

**Review by rafaljanwojcik — COMMENTED** [review]
Really nice PR, only small 2 comments

File: Makefile (hunk 2 above)
Comment (murielhol) [reply Makefile]: ah yes, good catch! I left it out to monitor the command line, will add it back in

File: document_indexing/populate_document_store.py (hunk 1 above)
Comment (murielhol) [reply document_indexing/populate_document_store.py]: sorry, its fixed as you can see :)

File: Makefile (hunk 2 above)
Comment (murielhol) [reply Makefile]: https://github.com/rafaljanwojcik/board-game-rules-explainer/pull/1/commits/93110fe7754538d6c2aa360f93182698cf19236f

**Review by rafaljanwojcik — APPROVED** [review]
Approve :)

**Pull request merged**